### PR TITLE
CVE-2010-2653

### PR DESCRIPTION
**Pull request by akn4743**

### CODE REVIEW

File: cves/CVE-2010-2652.yml [hunk 1]
```diff
@@ -32,10 +32,12 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description: "This is an availability vulnerability. It has to do with a web dialog\nnot
-  being handled properly. Creating strange delete behavior. The dialog \nhas been
-  queued, but it hasn't been shown because a different dialog is \nalready showing.
-  This causes a crash and disrupts availability.\n"
+description: |
+  This vulnerability pertains to the availability of the browser. If it was
+  exploited, the chromium web browser would crash. How this vulnerability
```
Comment (CatherineOsadciw) [inline cves/CVE-2010-2652.yml]: Reword to be a little more professional: cut the "how this vulnerability works is that" part.

File: cves/CVE-2010-2652.yml [hunk 2]
```diff
@@ -115,12 +116,8 @@ subsystem:
...
     directory names. Look at comments in the code. Look at the bug reports how
     the bug report was tagged. Examples: "clipboard", "gpu", "ssl", "speech", "renderer"
-  answer: 'Browser
-
-'
-  name: 'Browser
-
-'
+  answer: The vulnerability was found in /browser/app_modal_dialog_gtk.cc
+  name: Browser
```
Comment (CatherineOsadciw) [inline cves/CVE-2010-2652.yml]: Maybe add "Modal Dialog" as a subsystem?

File: cves/CVE-2010-2652.yml [hunk 3]
```diff
@@ -130,7 +127,9 @@ interesting_commits:
     emerging themes?
 
     If there are no interesting commits, demonstrate that you completed this section by explaining what happened between the VCCs and the fix.
-  answer:
+  answer: |
+    There is over a year of time between when the vulnerability was introduced and fixed. Between then, only once was the file that contained
```
Comment (CatherineOsadciw) [inline cves/CVE-2010-2652.yml]: Add some brief summaries of those in-between events to the "major events" section below

Comment (akn4743) [reply cves/CVE-2010-2652.yml]: Theres over 45,000 commits between the fix and the vcc. I'm adding a few events that occurred during that time period in the major events section but there is just too many commits to go through them all.

File: cves/CVE-2010-2652.yml (hunk 1 above)
Comment (akn4743) [reply cves/CVE-2010-2652.yml]: Made changes to the description

File: cves/CVE-2010-2652.yml [hunk 4]
```diff
@@ -32,10 +32,12 @@ description_instructions: |
...
-description: "This is an availability vulnerability. It has to do with a web dialog\nnot
-  being handled properly. Creating strange delete behavior. The dialog \nhas been
-  queued, but it hasn't been shown because a different dialog is \nalready showing.
-  This causes a crash and disrupts availability.\n"
+description: |
+  This vulnerability pertains to the availability of the browser. If it was
+  exploited, the chromium web browser would crash. Websites can create and allow 
+  the user to view Javascript dialogs. These dialogs can be queued so that another 
+  may pop up when one closes. The crash would occur when a dialog that is in the queue 
+  closes.
```
Comment (CraftSpider) [inline cves/CVE-2010-2652.yml]: How did one actually close a dialog that was still in the queue?

File: cves/CVE-2010-2652.yml [hunk 5]
```diff
@@ -148,10 +149,12 @@ major_events:
...
     wide change.
   events:
-  - name: Fixing the issue by adding a check to each call
+  - name: Enable chrome to display all cookies used/blocked on the current site.
     date: 2010-06-28
-  - name: 
-    date: 
+  - name: A large chunk of CVEs were being patched at the time of the fix.
+    date: 2010-07-02
+  - name: A throbber is being worked rewritten due to performance issues.
```
Comment (CraftSpider) [inline cves/CVE-2010-2652.yml]: What is a 'throbber'? This might use a more descriptive name

File: cves/CVE-2010-2652.yml [hunk 6]
```diff
@@ -57,10 +59,10 @@ fixes:
 - :commit: 66711f6f7abbf3ed9f0d39e4f497620ab39e79f0
   :note: ''
```
Comment (CraftSpider) [inline cves/CVE-2010-2652.yml]: What did this commit do to fix the issue?

**Pull request merged**